dsl/pace/dsl/dace/dace_config.py

```diff
@@ -183,24 +274,24 @@ def __init__(
 
         self._backend = backend
         self.tile_resolution = [tile_nx, tile_nx, tile_nz]
-        from pace.dsl.dace.build import get_target_rank, set_distributed_caches
+        from pace.dsl.dace.build import set_distributed_caches
```

Comment:
Can we please not import in the __init__ method?

Reply:
I wish but there's a cycling dependency. Back to what I was saying about needing a refactor to uproot all the CompileConfig/Build stuff into one central spot. And this will have a knock on effect on the DaCeConfig as well

Reply:
I'd rather not uproot one function into a single file - knowing that it's far from being the correct answer